test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+describe CategoriesController do
+  describe'create' do
+    let(:merchant) { Merchant.first }
+    before do
+      login(merchant)
```

Comment:
If creating a category requires you to be logged in, you should include a test where a guest user creates one and is rejected.